tests/infra/ledger.py

```diff
@@ -47,6 +47,7 @@ def __init__(self, buffer):
         self._buffer_size = buffer.getbuffer().nbytes
         self._unpacker = msgpack.Unpacker(self._buffer, raw=True, strict_map_key=False)
         self._version = self._read_next()
+        self._tables = {}
```

Comment:
This is the fix.

Reply:
Can we remove the class-static variables in this file? It appears they're defaults rather than usefully static, and our coding style elsewhere is to initialise/declare all member variables in `__init__`.

Reply:
Good point. Some of them were unused too.